### ISSUE TITLE
Demo animations in README

### ISSUE THREAD

**Issue by untitaker** [pimutils/todoman#81]
It would be nice to have a picture or even smaller audio-less demos in the README. Latter could be recorded with http://asciinema.org/

**Comment by hobarrera** [comment]
+1 It makes a big difference, see, for example, [pipenv](https://github.com/kennethreitz/pipenv/#pipenv-sacred-marriage-of-pipfile-pip--virtualenv).

**Comment by hobarrera** [comment]
So here's on I made today quickly:

    https://asciinema.org/a/c4oy6bm9axb8qyonparjmh1w2

Honestly, I'll completely open to criticism and a new script for it, etc. I'm not very creative with making up actions.

**Comment by untitaker** [comment]
A few things:

- ``--due`` (particularly its human datetime parsing) should be shown off
- A typo in your task description: "by" instead of "be".

otherwise LGTM

**Comment by untitaker** [comment]
https://asciinema.org/a/5gc9baoqtx6wkgi1ytv90ncrs and https://asciinema.org/a/db3xuhufirhq0g4b22jwmmjeg

**Comment by untitaker** [comment]
Uh oh, the latter actually revealed a bug in todoman. `-dtomorrow` doesn't work properly outside of UTC

**Comment by hobarrera** [comment]
Yeah, I also uncovered another bug (when not passing any summary) when doing this demo.

We need to figure out how to test the `urwid` bit.

**Comment by untitaker** [comment]
Turns out the default due date got set after setting the user-defined due date 💩 Fixed in masterr

**Comment by untitaker** [comment]
Here's another screencast: https://asciinema.org/a/7ifwbqub2f9e3ntgzk0iszf8g

**Comment by hobarrera** [comment]
Back on-topic:

* asciinema records your terminal size, so something more convervative (25*80) looks better. Especially if it goes into the README. I use i3wm, and just make the terminal window float for it to take its default size.
* I like showing off the config and `find`, but I'd move that to the end, and start off showing the app itself first.
* I mostly liked this one: https://asciinema.org/a/5gc9baoqtx6wkgi1ytv90ncrs
* I'd prefer "real-looking" todos. Rather than "hello world", make up something that sounds like an actual task.

Also of importance: I don't see how to download a video/gif from asciinema.

**Comment by untitaker** [comment]
I think we can edit the JSON recording just for resizing the terminal, but I'm not sure.

I suppose we have to record this in some other way because of the GIF though. A real video with sound would be nice

**Comment by hobarrera** [comment]
Sound?

**Comment by untitaker** [comment]
Something like the video at http://beets.io/

**Comment by hobarrera** [comment]
The voiceover is actually quite helpful, and showing the config makes for sense with it.

**Comment by hobarrera** [comment]
`ttyrec` and `ttygif` produce really good results. :+1:

**Comment by hobarrera** [comment]
Script idea:

* Start of with two tasks, with and without due dates, and lists with colors.
* `todo new SOMETHING`
* `todo new -l work SOMETHING`
* `todo`
* `todo edit 3`, edit due date
* `todo`
* `todo done 2`
* cat config
* tree paths
* cat ics file

More details:
* Make sure we use the latest version of `todoman` (unlike my previous recording).
* 80x25 terminal.
* Tasks are better if they sound real ("purchase ticket", "organize party", etc).
* I prefer `date_format = %Y-%m-%d %H:%M` for the demo.

**Comment by hartwork** [comment]
Maybe this issue is without progress for 7 years because the bar is too high. It doesn't take a well-scripted movie. Even a few screenshots may help for a start. Just my two cents.
